### PR TITLE
Add Datetime.timezone

### PR DESCRIPTION
**Pull request by k4nar**
This should be pretty much all of them. I've extracted them from https://timezonedb.com/time-zones.

### CODE REVIEW

File: mimesis/providers/date.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from calendar import monthrange
 
 from mimesis.data import ROMAN_NUMS
+from mimesis.data import TIMEZONES
```
Comment (lk-geimfari) [inline mimesis/providers/date.py]: Please, use following format:
```
from mimesis.data import ROMAN_NUMS, TIMEZONES
```

**Comment by lk-geimfari** [comment]
@k4nar Thank you! Please, look at very small change request.

**Comment by lk-geimfari** [comment]
@k4nar What do you think about GMT offset. Can it be useful? I mean following: 

```python
>>> Datetime().gmt_offset()
'UTC +02:00'
```

**Comment by lk-geimfari** [comment]
I'll fix it by myself, thank you.

**Comment by k4nar** [comment]
> I'll fix it by myself, thank you.

Ok great, I was about to do it :) .

> What do you think about GMT offset. Can it be useful?

I don't remember having this need before, but some might find it useful. However for a random timezone-aware datetime, I would advise to generate a datetime in UTC and switch to a random timezone with an appropriate library.

**Pull request merged**